store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/GetAllGraphIds.java

```diff
@@ -53,6 +48,7 @@ public GetAllGraphIds() {
     public GetAllGraphIds shallowClone() throws CloneFailedException {
         return new Builder()
                 .options(options)
+                .userRequestingAdminUsage(userRequestingAdminUsage)
```

Comment:
Missing graphIds from shallowClone, add a parent test and check All iFederationOperation